app/controllers/home-office-details.ts

```diff
@@ -1,20 +1,24 @@
 import { NextFunction, Request, Response, Router } from 'express';
+import _ from 'lodash';
 import moment from 'moment';
 import multer from 'multer';
-import i18n from '../../locale/en.json';
 import { paths } from '../paths';
 import { Events } from '../service/ccd-service';
 import UpdateAppealService from '../service/update-appeal-service';
 import { dateLetterSentValidation, homeOfficeNumberValidation, textAreaValidation } from '../utils/fields-validations';
+import { getConditionalRedirectUrl } from '../utils/url-utils';
 
 function getHomeOfficeDetails(req: Request, res: Response, next: NextFunction) {
   try {
+    req.session.isEdit = _.has(req.query, 'edit');
```

Comment:
isEdit shouldnt be inside appeal.application type?

Reply:
Yes, I agree, I'll amend on next commit. 
Thanks, @t0nin0s 👍